assets/images/logo/logo.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg viewBox="6.998 16.33 500 500" width="500" height="500" xmlns="http://www.w3.org/2000/svg">
+  <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="matrix(1.866706, 0, 0, 1.866706, 47.740799, 47.424915)" style="">
+    <g id="Master">
+      <g id="Logo" transform="translate(20.000000, 20.000000)">
+        <mask id="mask-2" fill="white">
+          <rect x="0" y="0" width="152" height="152" rx="12" transform="matrix(1, 0, 0, 1, 0, 0)"/>
+        </mask>
+        <rect id="Mask" x="0" y="0" width="152" height="152" transform="matrix(1, 0, 0, 1, 0, 0)" fill="#2879FF" rx="23.044" ry="23.044"/>
+        <g id="Grid" mask="url(#mask-2)" stroke="#FFFFFF" stroke-opacity="0.2" stroke-width="0.5">
+          <g transform="translate(-20.000000, -20.000000)">
+            <path d="M 0 0 L 192 192" id="Shape"/>
+            <path d="M 192 0 L 0 192" id="Shape"/>
+            <path d="M 123.75 0 L 123.75 192" id="Shape"/>
+            <path d="M 68.25 0 L 68.25 192" id="Shape"/>
+            <path d="M 192 123.75 L 0 123.75" id="Shape"/>
+            <path d="M 192 68.25 L 0 68.25" id="Shape"/>
+            <circle id="Oval" cx="96" cy="96" r="39"/>
+            <path d="M 0 96 L 192 96" id="Shape"/>
+            <path d="M 96 0 L 96 192" id="Shape"/>
+          </g>
+        </g>
+        <path d="M 140 0 L 12 0 C 5.4 0 0 5.4 0 12 L 0 13 C 0 6.4 5.4 1 12 1 L 140 1 C 146.6 1 152 6.4 152 13 L 152 12 C 152 5.4 146.6 0 140 0 Z" id="Shadow-Top" fill="#FFFFFF" opacity="0.2"/>
+        <path d="M 140 151 L 12 151 C 5.4 151 0 145.6 0 139 L 0 140 C 0 146.6 5.4 152 12 152 L 140 152 C 146.6 152 152 146.6 152 140 L 152 139 C 152 145.6 146.6 151 140 151 Z" id="Shadow-Bottom" fill="#000000" opacity="0.1"/>
+      </g>
+    </g>
+  </g>
+  <g style="" transform="matrix(13.518198, 0, 0, 13.518198, 123.816666, 124.260933)">
+    <path fill-rule="evenodd" d="M 10.96 4.446 C 10.463 3.425 10.993 2.281 12.075 2.036 C 12.956 1.836 13.868 2.499 13.985 3.415 C 14.036 3.818 13.958 4.187 13.763 4.533 C 12.158 7.388 10.55 10.242 8.947 13.099 C 8.649 13.629 8.216 13.951 7.622 13.995 C 6.957 14.044 6.434 13.757 6.098 13.166 C 5.082 11.377 4.078 9.58 3.069 7.787 C 2.455 6.695 1.839 5.605 1.228 4.512 C 0.612 3.41 1.307 2.07 2.544 2.005 C 3.197 1.971 3.701 2.279 4.032 2.859 C 4.485 3.653 4.929 4.452 5.378 5.249 C 5.701 5.822 6.018 6.4 6.35 6.968 C 6.832 7.792 7.539 8.257 8.481 8.315 C 9.814 8.396 11.051 7.408 11.211 5.957 C 11.223 5.848 11.231 5.74 11.236 5.684 C 11.228 5.215 11.143 4.819 10.96 4.446 Z" style="fill: rgb(255, 255, 255);"/>
+  </g>
+</svg>
```

Comment:
Missing EOL@EOF (end of line at end of file) 😉

Reply:
Sorry, but it was auto-generated. I can fix it though.